src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/IS91DeterminationServiceTest.kt

```diff
@@ -0,0 +1,81 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.reset
+import org.mockito.kotlin.whenever
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonApiClient
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonerOffenceHistory
+
+class IS91DeterminationServiceTest {
+  private val prisonApiClient = mock<PrisonApiClient>()
+  private val service = IS91DeterminationService(prisonApiClient)
```

Comment:
meh.. benefit of having this as a dependency is that you don't necessarily need to test its functionality here and can mock it out. 

You can just write tests against its contract, what happens if it filters bookings in or out - otherwise you're testing the same thing at multiple levels.

Probably not worth changing but worth keeping in mind